### PR TITLE
Update smart_proxy_openscap to 0.6.1

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [ ] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec [hunk 1]
```diff
@@ -24,7 +24,7 @@ BuildRequires: ruby(release)
 Requires: ruby(rubygems)
 Requires: foreman-proxy >= 1.11.0
 Requires: crontabs
-Requires: rubygem(openscap) >= 0.4.3
+Requires: rubygem(openscap) >= 0.4.7
```
Comment (domcleal) [inline rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec]: This isn't available on EL7, the package must be updated first.

Comment (ares) [inline rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec]: https://github.com/theforeman/foreman-packaging/pull/1530

**Comment by ares** [comment]
Yeah, I think that was a mistake and will be hopefully fixed in new version, anyway I added < 0.5.0 to match it correctly.

**Comment by domcleal** [comment]
Merged and building, thanks @ares.

**Pull request merged**